1.18/Dockerfile

```diff
@@ -59,10 +59,10 @@ RUN yum install -y yum-utils gettext hostname && \
     yum -y clean all --enablerepo='*'
 
 # Copy the S2I scripts from the specific language image to $STI_SCRIPTS_PATH
-COPY $NGINX_VERSION/s2i/bin/ $STI_SCRIPTS_PATH
+COPY 1.18/s2i/bin/ $STI_SCRIPTS_PATH
 
 # Copy extra files to the image.
-COPY $NGINX_VERSION/root/ /
+COPY 1.18/root/ /
```

Comment:
We have to find out, why NGINX_VERSION variable can not be used. I would not hardcoded.

Reply:
Kind of agree. The PR can be a partial step in fixing this.

Reply:
It was agreed, that using the hardcoded version in here is correct option.